src/domain/pageHeader/PageHeaderContainer.tsx

```diff
@@ -29,7 +29,15 @@ const PageHeaderContainer: React.SFC = () => {
       <Dropdown label={<Button icon={<Icon name="user" />}>{fullName}</Button>}>
         <List noBullets>
           <ListItem>
-            <Button variant="text" onClick={() => dispatch(userSignedOut())}>
+            <Button
+              variant="text"
+              onClick={() => {
+                // Flush data in redux store and localStorage
+                persistor.flush();
```

Comment:
Are we really persisting the data in the first place? I think we are not.

Reply:
This is an intermediary solution which I would like to refactor in the long run. For now, we are using `redux-persist` package. I haven't properly tested this and would like to get rid of this all together.

Reply:
I looked into it, it's safe to say that we can get rid of it now (unless we opt to persist the login state). I believe this `redux-persist` has leaked to our code from another service, and it is not used at the moment.

Reply:
ok